<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_sql_login</class>
 <widget class="QMainWindow" name="form_sql_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>322</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tài khoản đăng nhập</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_login_user">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sql_dangky_esare">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Xóa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_login_pass">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mật khẩu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên đăng nhập</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sql_dangky_exit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sql_dangky_add">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Đăng ký</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
